src/Database/Driver/Sqlserver.php

```diff
@@ -64,8 +65,14 @@ public function connect()
             return true;
         }
         $config = $this->_config;
+
+        // PDO::ATTR_PERSISTENT is not allowed by SQL Server PHP drivers.
+        // See: https://github.com/Microsoft/msphpsql/issues/65
+        if (isset($config['persistent']) && $config['persistent']) {
+            throw new InvalidSettingException(['setting' => 'persistent']);
```

Comment:
Not sure we need an exception class for this. You could use InvalidArgumentException instead.